rust/private/rustc.bzl

```diff
@@ -819,6 +820,7 @@ def construct_arguments(
         build_metadata (bool): Generate CLI arguments for building *only* .rmeta files. This requires use_json_output.
         force_depend_on_objects (bool): Force using `.rlib` object files instead of metadata (`.rmeta`) files even if they are available.
         skip_expanding_rustc_env (bool): Whether to skip expanding CrateInfo.rustc_env_attr
+        sandboxed (bool): Whether the action using the constructed arguments is sandboxed
```

Comment:
This seems like a weird name. I would have expected to see `add_sysroot` or something. Can you explain what the relevance of sandboxing is here?

Reply:
Sorry, I was trying to make sure `rustdoc`'s implementation doesn't change. We shouldn't need this if we use `toolchain. sysroot`. I removed it.